BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmyViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  JimmyViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+class JimmyViewController: UIViewController, canRecieve {
+
+    @IBOutlet var label: UILabel!
+    @IBOutlet var textField: UITextField!
+    
+    override func viewDidLoad() {
+
+
+    }
+    
+    
+    @IBAction func buttonPressed(_ sender: Any) {
+        performSegue(withIdentifier: "sendDataForwards", sender: self)
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "sendDataForwards" {
+            
+            let secondVC = segue.destination as! JimmySecondViewController
+            
+            secondVC.data = textField.text!
+            
+            secondVC.delegate = self
+        }
+    }
+    
+    func dataRecieved(data: String) {
```

Comment:
This is totally fine, though I am a fan of creating a separate extension that conforms to the delegate protocol and have the delegate methods there for tidiness.

Reply:
Agreed, this is pretty much a community standard. It should go at the end of the VC something like this:
```
extension JimmyViewController: JimmySecondViewController {

}
```